lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/metrics/TDigestHistogram.java

```diff
@@ -52,18 +48,30 @@ public static TDigestHistogram fromValue(double value, double weight)
         return new TDigestHistogram(digest);
     }
 
-    @JsonCreator
     public TDigestHistogram(TDigest digest)
     {
         this.digest = digest;
     }
 
-    @JsonProperty
+    @JsonIgnore
```

Comment:
This shouldn't be needed. Anything not annotated with `@JsonProperty` should not be being included by Jackson.

Reply:
Removed.